### PR TITLE
Morton2000

### PR DESCRIPTION
**Pull request by profxj**
Added Morton2000 and made a few other edits.
Added tests for reading Morton files.
Changed Verner1994 to be a FITS table.

### CODE REVIEW

File: linetools/lists/parse.py [hunk 1]
```diff
@@ -150,6 +151,13 @@ def read_verner94():
     vot = vo_parse(verner94)
     # Table
     tbl_6 = vot.get_table_by_index(5).to_table(use_names_over_ids=True)
+    '''
```
Comment (nhmc) [inline linetools/lists/parse.py]: Can you remove this instead of commenting out?

**Comment by nhmc** [comment]
All the notebook examples should (eventually) be moved to docs/examples/. Otherwise this looks fine to merge, when you're ready.

File: linetools/lists/parse.py (hunk 1 above)
Comment (profxj) [inline linetools/lists/parse.py]: Gone.

**Comment by profxj** [comment]
Done.  Started an doc/examples folder.
Pushed both notebooks in.
Will push all further example files there.

**Pull request merged**